### PR TITLE
Show line style in locked line summary

### PR DESCRIPTION
**Pull request by nishasy**
## Summary:
Create a LineSwatch component so we can show the line styles
in the LockedLineSettings summary.

[Design convo here](https://khanacademy.slack.com/archives/C067UM1QAR4/p1715280767285859)

Issue: none

## Test plan:
`yarn jest`

Storybook
- http://localhost:6006/?path=/docs/perseuseditor-components-locked-line-settings--docs
- http://localhost:6006/?path=/story/perseuseditor-editorpage--mafs-with-locked-figures

| Solid | Dashed |
| --- | --- |
| Screenshot 2024-05-14 at 12 05 24 PM | Screenshot 2024-05-14 at 12 05 18 PM |

### CODE REVIEW

File: packages/perseus-editor/src/components/line-swatch.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const styles = StyleSheet.create({
+    container: {
+        backgroundColor: wbColor.white,
+        justifyContent: "center",
+        padding: spacing.xSmall_8,
+        borderRadius: spacing.xxxSmall_4,
+    },
+    lineSwatch: {
+        // Add a white outline so that the color swatch is visible when
+        // the dropdown option is highlighted with its blue background.
```
Comment (jeremywiebe) [inline packages/perseus-editor/src/components/line-swatch.tsx:39]: This doesn't seem related to this style property. 🤔   Should it be in the `container` style?

**Review by jeremywiebe — APPROVED** [review]
Really like this new visual!

File: packages/perseus-editor/src/components/line-swatch.tsx (hunk 1 above)
Comment (nishasy) [reply packages/perseus-editor/src/components/line-swatch.tsx:39]: Oh! Oops, I think that's left over from when I copied this from the color swatch

Comment (nishasy) [reply packages/perseus-editor/src/components/line-swatch.tsx:39]: I didn't push before landing, so now it's still there in the code 😑
